testing/btest/scripts/base/files/x509/log-caching-cluster.test

```diff
@@ -5,6 +5,7 @@
 # @TEST-PORT: BROKER_WORKER2_PORT
 #
 # @TEST-EXEC: cp $FILES/broker/cluster-layout.zeek .
+# @TEST-EXEC: zeek --parse-only %INPUT
```

Comment:
Is this here for a reason, or is this a leftover from debugging?

Reply:
I've started to put them in a few cluster tests because the `btest-bg-run` / `btest-bg-wait` sequence doesn't fail quickly on syntax errors while working on the tests (or later when debugging a flaky test).

I'll drop them. It's kind of my own approach to working with the tests.